### ISSUE TITLE
centos7 build RedisSearch error sed: -e expression #1, char 30: unterminated `s' command

### ISSUE THREAD

**Issue by caoxiaoyuan7242** [RediSearch/RediSearch#2177]
sudo make setup throw an exception:sed: -e expression #1, char 30: unterminated `s' command
file：deps/readies/bin/enable-utf8
runn sed -i 's/^\(override_install_langs=\)/# \1/' /etc/yum.conf
![image](https://user-images.githubusercontent.com/33684848/129027272-cd11448b-624b-4513-8433-de64a60b810e.png)

**Comment by rafie** [comment]
Hello @caoxiaoyuan7242, thank you for reporting.
Would you care to share your CentOS/RHEL and sed versions?
Thanks!

**Comment by caoxiaoyuan7242** [comment]
hello @rafie CentOS 7.6.1810 sed (GNU sed) 4.2.2

**Comment by ficus** [comment]
Hi, ran into this as well on centos7. I build gnu make 4.3 as a prerequisite and then use it for the setup and build of redisearch.  I pulled your changes from `readies` into deps of redisearch as I did not see them in redisearch yet. I was then able to build successfully.  Thanks for that fix. This was a fresh AWS image, centos7 HVM.

Is setup of cmake3 part of this code? I'd note that `Setup` worked, but before I could get `make4 build` to complete, I had to reinstall cmake3, or it might be I needed to restart the shell for it to be on the path? Error suggests cmake must be greater than version 3 but think `which cmake` was actually failing until I yum reinstall / exit & reconnect. I've experienced this in the past as well - something weird there if it is one of the prerequisites this tries to install.
